accord-core/src/test/java/accord/primitives/TxnIdTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package accord.primitives;
+
+import org.junit.jupiter.api.Test;
+
+import accord.utils.AccordGens;
+import org.assertj.core.api.Assertions;
+
+import static accord.utils.Property.qt;
+
+class TxnIdTest
+{
+    @Test
+    void stringSerde()
+    {
+        qt().forAll(AccordGens.txnIds()).check(id -> {
+            Assertions.assertThat(TxnId.parse(id.toString())).isEqualTo(id);
+
+            Assertions.assertThat(Timestamp.fromString(id.toStandardString())).isEqualTo(new Timestamp(id));
```

Comment:
this is the test referenced above... `Timestamp.fromString()` for a txn wouldn't work with `toString` so need a `toStandardString`